Fakebook.Profile/Fakebook.Profile.RestApi/Controllers/ProfileController.cs

```diff
@@ -162,5 +162,46 @@ public async Task<ActionResult> UpdateAsync([FromBody] ProfileApiModel apiModel)
                 return BadRequest();
             }
         }
+
+        [HttpPost("follow/{user=user}")]
```

Comment:
why is there a route parameter here defaulting to "user"?
i don't think we need to post the whole profile - just the email in the URL should be enough, no?

Reply:
Changed to a user email instead of entire user profile